### PR TITLE
tasks-update

### PR DESCRIPTION
**Pull request by danielseremet**
final?

### CODE REVIEW

File: src/main/java/com/learn/task1/Task1.java [hunk 1]
```diff
@@ -14,18 +11,37 @@ public Task1(List<Auto> autos) {
     }
 
     public List<Auto> filterByPrice(double minPrice) {
-        return autos.stream().filter(auto -> auto.getPrice() >= minPrice).collect(Collectors.toList());
+        return autos.stream().filter(auto -> auto.getPrice() >= minPrice).toList();
```
Comment (LunguIon) [inline src/main/java/com/learn/task1/Task1.java:14]: sonarlint installed, good

**Review by LunguIon — CHANGES_REQUESTED** [review]
I don't see you solving other issues, as I specified in the last PR, and when speaking with you at the gym, where are the tests for edge cases ? What happens if the list empty, or we have only one element in the list, or we have cars with the same price ??? Please address this issues and update the PR
